tlsutil/config.go

```diff
@@ -30,8 +30,12 @@ var TLSLookup = map[string]uint16{
 	"tls10": tls.VersionTLS10,
 	"tls11": tls.VersionTLS11,
 	"tls12": tls.VersionTLS12,
+	"tls13": tls.VersionTLS13,
 }
 
+// TLSVersions has all the keys from the map above, make sure to keep both in sync
+var TLSVersions = "tls10, tls11, tls12, tls13"
```

Comment:
Couldn't we instead use an `init` func and programmatically generate this?

Reply:
Good catch! I went with a normal function, instead. But the idea is the same.